tests/functional/test_zilien_parser.py

```diff
@@ -0,0 +1,146 @@
+"""This module perform top-level finctional tests on the Zilien parsers"""
+
+import datetime
+import time
+from collections import namedtuple
+from pathlib import Path
+
+import pytest
+from pytest import approx
+from ixdat.techniques.ec_ms import ECMSMeasurement, MSMeasurement, ECMeasurement
+
+DATA_DIR = Path(__file__).parent / ".." / ".." / "test_data"
```

Comment:
why not `Path(__file__).parent.parent.parent / "test_data"`?

Reply:
I really don't know :D Fixed